src/components/NavigationGroup.svelte

```diff
@@ -1,6 +1,6 @@
 {{#if separator == 'top'}}<hr />{{/if}}
 {{#if group.length}}
-<ul class='coz-nav-group'>
+<ul class='{{`coz-nav-group${wrappingClass}`}}'>
```

Comment:
ah yeaaaah ok. In that case, i'd rather see a `coz-nav-group--wrapping` to be more consistent cause i don't see any class in Bar's CSS beginning with `--`, and it's kinda confusing.